### PR TITLE
Cache stats

### PR DESCRIPTION
**Pull request by pokey909**
@tchakabam @milos-pesic-sc please take a look. Needed for SKIP-263

### CODE REVIEW

**Comment by milos-pesic-sc** [comment]
:+1:

File: src/skippy_uridownloader.c [hunk 1]
```diff
@@ -650,3 +650,22 @@ skippy_uri_downloader_fetch_fragment (SkippyUriDownloader * downloader, SkippyFr
   GST_OBJECT_UNLOCK (downloader);
   return SKIPPY_URI_DOWNLOADER_COMPLETED;
 }
+
+void
+skippy_uri_downloader_bytes_received_hook(SkippyUriDownloader * downloader, gsize bytes_loaded) {
+  gboolean is_read_from_cache = FALSE;
+  g_object_get(G_OBJECT(downloader->priv->urisrc), "cache-read-mode", &is_read_from_cache, NULL);
```
Comment (tchakabam) [inline src/skippy_uridownloader.c]: you should if the object has that property first. this is a custom HTTP src thing, the ones that come with gst plugins by default don't have it ;)

**Comment by tchakabam** [comment]
Ok great, except the comment above that needs an additional check it's cool :+1: 

It's going to be cake doing the exact same thing with the refactored downloader :)

**Comment by tchakabam** [comment]
however see my comment on the main repo

**Comment by pokey909** [comment]
Not needed anymore. closing.

**Pull request closed without merging**